linebot/models/responses.py

```diff
@@ -473,3 +473,35 @@ def __init__(self, request_id=None, **kwargs):
         super(NarrowcastResponse, self).__init__(**kwargs)
 
         self.request_id = request_id
+
+
+class AudienceGroup(Base):
+    """AudienceGroups.
+
+    https://developers.line.biz/en/reference/messaging-api/#get-audience-group
+    """
+
+    def __init__(self, audience_group_id=None, type=None, description=None, status=None,
```

Comment:
[must]
`AudienceGroup` should contain `jobs` property.
https://developers.line.biz/en/reference/messaging-api/#response-26

Reply:
Completed